spiketoolkit/validation/quality_metric_classes/utils/curationsortingextractor.py

```diff
@@ -236,9 +236,35 @@ def split_unit(self, unit_id, indices):
             self._roots.append(new_root_2)
 
             for feature_name in self.get_unit_spike_feature_names(unit_id):
-                full_features = np.array(self.get_unit_spike_features(unit_id, feature_name))
-                self.set_unit_spike_features(new_root_1_id, feature_name, full_features[indices_1])
-                self.set_unit_spike_features(new_root_2_id, feature_name, full_features[indices_2])
+                if feature_name.endswith('_idxs'):
+                    continue
+                if isinstance(full_features, list):
+                    full_features = np.array(full_features)
```

Comment:
this fails as `full_features` is not defined.
You should add this line before:
`full_features = self.get_unit_spike_features(unit_id, feature_name)`

Reply:
@llobetv

Reply:
I was too fast sorry, fix :)